packages/aws-cdk-lib/aws-lambda/README.md

```diff
@@ -1143,6 +1143,27 @@ const fn = new lambda.Function(this, 'Lambda_with_IPv6_VPC', {
 });
 ```
 
+## Outbound traffic
+By default, when creating a Lambda function, it would add a security group outbound rule to allow sending all network traffic (except IPv6). This is controlled by `allowAllOutbound` in function properties, which has a default value of `true`.
+
+To allow outbound IPv6 traffic by default, explicitly set `allowAllIpv6Outbound` to `true` in function properties as shown below (the default value for `allowAllIpv6Outbound` is `false`):
+```ts
+const vpc = new ec2.Vpc(this, 'Vpc', {
+  maxAzs: 2,
+  restrictDefaultSecurityGroup: false,
```

Comment:
Is `restrictDefaultSecurityGroup: false` required if we need to allow all outbound traffic? If yes, maybe we should note this in the README? If not, let's remove this from the sample?

Reply:
Removed